src/holon/models/rule_mapping.py

```diff
@@ -1,12 +1,9 @@
-import sentry_sdk
-
 from holon.models.scenario import Scenario
 from holon.models.scenario_rule import ScenarioRule
 from holon.serializers import InteractiveElementInput
 from holon.models import ChoiceType
 
 
-@sentry_sdk.trace
 def get_scenario_and_apply_rules(
```

Comment:
Vergeten de nieuwe decorator voor sentry weer toe te voegen, of hoeft hier geen sentry logging op?

Reply:
Check